### PR TITLE
Default State final test

### PR DESCRIPTION
**Pull request by NinaNuzhna**
Test

### CODE REVIEW

File: test/specs/DefaultState.spec.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        expect(actualAddBtn3Text).toHaveText(expAddBtn3Text);
+    })
+
+    it('should DELETE button have text DELETE', () => {
+        const actualDeleteBtn = $('//button[@id="1"]');
+        expect(actualDeleteBtn).toHaveText('DELETE');
+    })
+
+    it('should Edit name field be present', () => {
+        expect($(EditNameField)).toBeDisplayed;
```
Comment (GalinaRuban) [inline test/specs/DefaultState.spec.js:43]: please change to toBeDisplayed() Not working properly without ()

File: test/specs/DefaultState.spec.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        expect(resetButton).toHaveText("RESET");
+    })
+
+    it('should "Enter Counter Title:" text be present', () => {
+        const enterCounterTitle = $('//label[contains(text(),"Enter Counter Title")]');
+        expect(enterCounterTitle).toHaveText("Enter Counter Title:");
+    })
+
+    it('should Add Name Field be present', () => {
+        expect($(addNameField)).toBeDisplayed;
```
Comment (GalinaRuban) [inline test/specs/DefaultState.spec.js:66]: please change to toBeDisplayed()

File: test/specs/DefaultState.spec.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        expect($(addNameField)).toHaveValue("Counter Name");
+    })
+
+    it('should "Enter Initial Count:" text be present', () => {
+        const enterInitialCount = $('//label[contains(text(),"Enter Initial Count")]');
+        expect(enterInitialCount).toHaveText("Enter Initial Count:");
+    })
+
+    it('should Default Value Field be present', () => {
+        expect($(defaultValueField)).toBeDisplayed;
```
Comment (GalinaRuban) [inline test/specs/DefaultState.spec.js:79]: please change to toBeDisplayed()

File: test/specs/DefaultState.spec.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    })
+
+    it('should Default Value Field contain "50" by default', () => {
+        expect($(defaultValueField)).toHaveValue("50");
+    })
+
+    it('should Add Counter Button be present', () => {
+        expect($(addCounterBtn)).toBeDisplayed;
```
Comment (GalinaRuban) [inline test/specs/DefaultState.spec.js:87]: please change to toBeDisplayed()

File: test/specs/DefaultState.spec.js (hunk 1 above)
Comment (NinaNuzhna) [reply test/specs/DefaultState.spec.js]: Corrected

File: test/specs/DefaultState.spec.js (hunk 2 above)
Comment (NinaNuzhna) [reply test/specs/DefaultState.spec.js]: Corrected

File: test/specs/DefaultState.spec.js (hunk 3 above)
Comment (NinaNuzhna) [reply test/specs/DefaultState.spec.js]: Corrected

File: test/specs/DefaultState.spec.js (hunk 4 above)
Comment (NinaNuzhna) [reply test/specs/DefaultState.spec.js]: Corrected

**Pull request merged**